### ISSUE TITLE
[Bug] Bug while applying dictionary correction English US Dictionary.

### ISSUE THREAD

**Issue by joaocarvalhoopen** [Zettlr/Zettlr#3353]
### Description

When I activate the English US dictionary, and have previously typed the following sentence:

each diretory I want to see all files.

It underlines the word diretory and correctly signals has an error, but when I follow the suggestion and apply it it eats the space between the word "directory" and the word "I" because to correct the error it has to add one more letter to the word and doesn't make space for it overriding the already existing space character.

### Reproducing

Ativate the EN-US dictionary

And amke a note with the following text

"each diretory I want to see all files."

Correct the "diretory" word and it will eat the space between "directory" and "I"

### Zettlr Version

Stable (most recent version)

### Specify version

2.2.4

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Ubuntu 20.04

### Additional Information

Thank you for all your work!

**Comment by joaocarvalhoopen** [comment]
I’m so sorry but **I can’t reproduce the bug anymore**, when I wrote the bug report it was happening inside a file with more text, but in the simple case that I extracted to the bug report when I put it into a simple file I’m not able to reproduce the bug. Not with my version 2.2.4 latest, nor with the development git clone version running with yarn test-gui. I was searching in the source code for the place where the text is replaced, but I have to study Vue.js because I lost myself a little bit in the events when the user clicks the popup menu with the suggestion words.

Best regards,
João
